otherlibs/systhreads/Makefile

```diff
@@ -22,7 +22,7 @@ ifneq "$(CCOMPTYPE)" "msvc"
 OC_CFLAGS += -g
 endif
 
-OC_CFLAGS += $(SHAREDLIB_CFLAGS)
+OC_CFLAGS += $(SHAREDLIB_CFLAGS) $(PTHREAD_CFLAGS)
```

Comment:
Review note: This change affects all configurations, but it also looks correct (the content of this variable is not included in some other FLAGS variable, so in general it would be missing here). I am actually surprised that not more systems need this.

Reply:
@shindere was working on CFLAGS - as far as I remember, it is yet to be complete. Perhaps he needs to take a look at this.

Reply:
This change is also not strictly necessary. There is a bit of background in the first comment of this PR.

Reply:
Chose the way of removing OS-specific bits from sources, so this change in Makefile will have to remain for Solaris.